### DISCUSSION TITLE
Porting Tensorflow Lite Edge - Fprime

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1398] [Q&A]
Hello,

I want to port and run the Tensorflow lite framework along with Fprime. 

In that case, I would like to run some of the edge math computations as the component. Can I run the Edge Component at the Driver framework level ?

**Comment by timcanham** [comment]
You can run it at any level. The "Driver framework level" is just a convention for defining layers in the architecture. When you define a component, you can decide to connect it to any component you like and at any priority you like if it is an active component.

**Comment by LeStarch** [comment]
You may wish to look into: https://github.com/LeStarch/fprime-python.  This has been used to run python code as part of a component in F´.  Note: is is research code at the moment and is not as mature as F´ itself.
